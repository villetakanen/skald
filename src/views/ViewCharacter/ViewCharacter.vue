<template>
  <v-container>
    <v-row>
      <v-column>
        <Character
          siteid=""
          playerid=""
          />
      </v-column>
    </v-row>
  </v-container>
</template>

<script lang="ts">
import Vue from 'vue'
import VueCompositionApi, { defineComponent } from '@vue/composition-api'
import Character from '../../components/character/Character.vue'

Vue.use(VueCompositionApi)

export default defineComponent({
  components: {
    Character
  }
})
</script>
