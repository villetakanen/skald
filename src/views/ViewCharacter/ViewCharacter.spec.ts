import { shallowMount } from '@vue/test-utils'
import ViewCharacter from './ViewCharacter.vue'

describe('Mounted App', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(ViewCharacter)
    expect(wrapper.element).toMatchSnapshot()
  })
})
