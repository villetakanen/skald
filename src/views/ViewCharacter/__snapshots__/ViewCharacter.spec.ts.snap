// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mounted App renders correctly 1`] = `
<v-container-stub
  tag="div"
>
  <v-row-stub
    tag="div"
  >
    <v-column>
      <character-stub
        playerid=""
        siteid=""
      />
    </v-column>
  </v-row-stub>
</v-container-stub>
`;
